data/json/items/comestibles/med.json

```diff
@@ -1167,6 +1167,29 @@
                 [ "calcium", 75 ]
             ]
         }
+    },
+        {
+        "id" : "bonemeal_tablet",
+        "type" : "COMESTIBLE",
+        "comestible_type" : "MED",
+        "name" : "bone meal tablet",
+        "description" : "Homemade calcium supplement made out of bone meal. Tastes horribly and is hard to swallow but it does its job.",
+        "weight" : 1,
```

Comment:
Make the weight 2g per pill and then they've got a realistic 50% calcium content.